chromadb/api/types.py

```diff
@@ -115,7 +116,8 @@ def validate_ids(ids: IDs) -> IDs:
         if not isinstance(id, str):
             raise ValueError(f"Expected ID to be a str, got {id}")
     if len(ids) != len(set(ids)):
-        dups = set([x for x in ids if ids.count(x) > 1])
+        c = Counter(ids)
```

Comment:
Thank you for doing this! Let's change the error message to just enumerate the first 10 duplicates as you suggested and then add ellipsis to truncate so that we aren't dumping potentially so many ids as to overload the console. Also we can do this in one pass if we store the call to set()